mmd_tools/panels/tool.py

```diff
@@ -31,6 +31,7 @@ def draw(self, context):
         col = layout.column(align=True)
         col.operator(operators.material.ConvertMaterialsForCycles.bl_idname, text='Convert Materials For Cycles')
         col.operator('mmd_tools.separate_by_materials', text='Separate By Materials')
+        col.operator(operators.misc.JoinMeshes.bl_idname)
 
```

Comment:
I think it's better to use bl_idname directly like `col.operator('mmd_tool.join_meshes')`, but it's fine...